src/spinnaker-dependencies.template

```diff
@@ -1,3 +1,4 @@
+# GENERATED FILE: ANY CHANGES WILL BE OVERWRITTEN
```

Comment:
Ahh, yes. I think it's easier to see what changes if we back that out. I'll do it.